web-components/src/components/dropdown/tokens/lm-dropdown-tokens.js

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable no-undef */
+const colors = require("@momentum-ui/tokens/dist/colors.json");
+
+const dropdown = {
+  prefix: "lm",
+  component: "dropdown",
+  "bg-color": {
+    light: colors.white[100].name,
+    dark: colors.gray[100].name
```

Comment:
Be sure to look at the global tokens first. Declaring new color tokens for the specific component is only necessary when the design has a unique outlier. Most hues are available already, for example this bg-color pair is already defined as `--md-primary-bg-color` in global. 
check `global--dark.scss` next to `global--light.scss` for existing pairs

Reply:
Ok